Debug App/Sources/View Controllers/MerchantSessionAndSettingsViewController.swift

```diff
@@ -905,7 +905,7 @@ class MerchantSessionAndSettingsViewController: UIViewController {
             paymentMethodOptions: PrimerPaymentMethodOptions(
                 urlScheme: "merchant://primer.io",
                 applePayOptions: PrimerApplePayOptions(
-                    merchantIdentifier: "merchant.checkout.team",
+                    merchantIdentifier: "merchant.dx.team",
```

Comment:
Why did we need this change in the team name?

Reply:
this was registered to our latest certificates